public/imgs/pole_black_yellow.svg

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
again this is personal preference but i like to rm verbosity so no need to label each one `black_yellow` if they all are